scipy/sparse/_base.py

```diff
@@ -119,7 +119,7 @@ def __init__(self, maxprint=MAXPRINT):
     def reshape(self, *args, **kwargs):
         """reshape(self, shape, order='C', copy=False)
 
-        Gives a new shape to a sparse array without changing its data.
+        Gives a new shape to a sparse array/matrix without changing its data.
```

Comment:
I think that this reads better. What do you think?
```suggestion
        Gives a new shape to a sparse array or matrix without changing its data.
```

This comment applies in many other places.

Reply:
I think both are fine, though I might slightly prefer "array/matrix".